src/utils/urls/appUrlHelper.ts

```diff
@@ -116,6 +116,7 @@ export const getLayoutSettingsUrl = (layoutset: string | null | undefined) => {
 };
 
 export const getLayoutSetsUrl = () => `${appPath}/api/layoutsets`;
+export const getFooterLayoutUrl = () => `${appPath}/api/footer`;
```

Comment:
A bit late to comment this, but some other apis use `/api/v1/<name>` which I think is better. We've already made `footer.schema.v1.json` so there might be a footer v2 in the future where we'd like to have another API endpoint as well.

Reply:
I can change that, app-lib PR is not merged yet